nanithing-web/pom.xml

```diff
@@ -1,159 +1,158 @@
+  <description>Nanithing: The web application</description>
 
-		<!-- SPECS -->
-		<dependency>
-			<groupId>javax.annotation</groupId>
-			<artifactId>javax.annotation-api</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>javax.enterprise</groupId>
-			<artifactId>cdi-api</artifactId>
-		</dependency>
-		
-		<!-- PROJECT -->
-		<dependency>
-			<groupId>nanithing</groupId>
-			<artifactId>nanithing-dao-jpa</artifactId>
-			<version>${project.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>nanithing</groupId>
-			<artifactId>nanithing-jaxrs-interfaces</artifactId>
-			<version>${project.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>nanithing</groupId>
-			<artifactId>nanithing-model</artifactId>
-			<version>${project.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>nanithing</groupId>
-			<artifactId>nanithing-services</artifactId>
-			<version>${project.version}</version>
-		</dependency>
-		
-		<!-- TEST -->
-		<dependency>
-			<groupId>org.jboss.resteasy</groupId>
-			<artifactId>resteasy-cdi</artifactId>
-			<exclusions>
-				<exclusion>
-					<groupId>org.jboss.weld</groupId>
-					<artifactId>weld-api</artifactId>
-				</exclusion>
-			</exclusions>
-		</dependency>
-		<dependency>
-			<groupId>org.jboss.resteasy</groupId>
-			<artifactId>resteasy-core</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.jboss.resteasy</groupId>
-			<artifactId>resteasy-json-binding-provider</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.jboss.weld</groupId>
-			<artifactId>weld-junit5</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.junit.jupiter</groupId>
-			<artifactId>junit-jupiter-engine</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.mockito</groupId>
-			<artifactId>mockito-core</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.mockito</groupId>
-			<artifactId>mockito-junit-jupiter</artifactId>
-		</dependency>
-	</dependencies>
-	
-	<build>
-		<resources>
-			<resource>
-				<directory>src/main/resources</directory>
-				<filtering>true</filtering>
-			</resource>
-		</resources>
+  <properties>
+    <jvm.args>-Dquarkus.datasource.url=${database.nanithing.url} -Dquarkus.datasource.username=${database.nanithing.username} -Dquarkus.datasource.password=${database.nanithing.password}</jvm.args>
+  </properties>
 
-		<plugins>
-			<plugin>
-				<groupId>io.quarkus</groupId>
-				<artifactId>quarkus-maven-plugin</artifactId>
-				<executions>
-					<execution>
-						<goals>
-							<goal>build</goal>
-						</goals>
-					</execution>
-				</executions>
-			</plugin>
-			<plugin>
-				<artifactId>maven-surefire-plugin</artifactId>
-				<configuration>
-					<systemProperties>
-						<java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
-					</systemProperties>
-				</configuration>
-			</plugin>
-		</plugins>
-	</build>
+  <dependencies>
+    <!-- QUARKUS -->
```

Comment:
The `nanithing-web` module is the one that actually depends on Quarkus. Add the `quarkus-smallrye-openapi` here. No need for version, it is managed from `quarkus-bom`.